### ISSUE TITLE
Feature Request: please add possibility to configure CAN filtering

### ISSUE THREAD

**Issue by Woodwarth** [arduino/ArduinoCore-renesas#162]
There is no possibility to set CAN Mask and Filter

### Additional context

Discussion thread:

https://forum.arduino.cc/t/uno-r4-can-mask-filter/1177947

**Comment by aentinger** [comment]
You are correct. I believe what you want to say is: "Feature Request: please add possibility to configure CAN filtering", right?

**Comment by Woodwarth** [comment]
new post in forum with working solution

**Comment by aentinger** [comment]
Great! How about providing a [pull request](https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) so that we can add your changes to this repository? I can assist you in case there are any problems 😉 .

**Comment by Woodwarth** [comment]
Hello,Never done this bevore, if you can say me how to do this, ill do it. 👌Am 02.11.2023 um 06:10 schrieb Alexander Entinger ***@***.***>:﻿
Great! How about providing a pull request so that we can add your changes to this repository? I can assist you in case there are any problems 😉 .

**Comment by per1234** [comment]
Closing as resolved by https://github.com/arduino/ArduinoCore-renesas/pull/170.

Thanks @Woodwarth!
